client/css/layout.css

```diff
@@ -266,31 +266,31 @@ body.aspectTooGood #hideToolbarButton {
 }
 
 #statesButton {
-  background-image:url(/i/icons-white/chess-solid-svg.svg);
+  background-image:url(./i/icons-white/chess-solid-svg.svg);
```

Comment:
Just curious: why `./i` instead of `i`?

Reply:
Honestly it's just for easier searching in the future. `./i/` is more unique than `i/`.